Topics/Software_Tools/PyAutoGUI/Automation_with_PyAutoGUI.md

```diff
@@ -166,7 +166,84 @@ pyautogui.doubleClick()
 ![sample_gif](./assets/move_folder.gif)
 
 ## Keyboard Controls
-Pending
+## Keyboard Controls
+Once familiar with mouse functions, exploring keyboard interactions becomes the next vital step. PyAutoGui offers various methods for keyboard input, enhancing automation capabilities.
```

Comment:
The brief introduction is great here. Consider adding a couple of example use cases in order to provide motivation behind the need to use the keyboard functionality.

Reply:
Agreed! Thats a great idea. I will add use cases catered towards CSC301 students.